src/main/java/uk/nhs/tis/trainee/notifications/service/NotificationService.java

```diff
@@ -170,6 +175,17 @@ public NotificationService(EmailService emailService, HistoryService historyServ
    * @return the result map with status details if successful.
    */
   public Map<String, String> executeNow(String jobKey, JobDataMap jobDetails) {
+    return messageSendingService.sendJobToOutbox(jobKey, jobDetails);
+  }
+
+  /**
+   * Process a queued job from the outbox.
+   *
+   * @param jobDetails The job details.
+   * @return the result map with status details if successful.
+   */
+  @SqsListener("${application.queues.outbox}")
+  public Map<String, String> executeQueued(JobDataMap jobDetails) {
```

Comment:
We probably need to check that the PM hasn't been deleted since its notification(s) were added to the queue

Reply:
I think we're getting into rework territory here, the future is probably that we queue notification IDs and pull latest from the DB instead of taking the quartz job data.